scripts/utils.py

```diff
@@ -226,7 +227,7 @@ def example(self, value: str):
         # Since we now have a new configuration, check the dataset and set the id
         self._data_set_id = self.verify_dataset()
 
-    def verify_client(self, capabilities: dict[list] = {}) -> CogniteClient:
+    def verify_client(self, capabilities: dict[str, list[str]] | None = None) -> CogniteClient:
```

Comment:
As discussed here, would be fantastic to get a generic verification into the SDK, but the current one here relying on the example config and preset data_set_id just makes problems for us. I'll leave it to your judgment on how much to do here vs work on the SDK side.